autoload/gitgutter/diff.vim

```diff
@@ -1,5 +1,12 @@
 let s:grep_available = executable('grep')
-let s:grep_command = ' | '.(g:gitgutter_escape_grep ? '\grep' : 'grep').' --color=never -e '.gitgutter#utility#shellescape('^@@ ')
+if s:grep_available
+  let s:grep_help = gitgutter#utility#system('grep --help')
+  let s:grep_command = ' | '.(g:gitgutter_escape_grep ? '\grep' : 'grep')
+  if s:grep_help =~# '--color'
```

Comment:
One final tiny thing: please could you swap lines 3 and 4?  The grep command is the primary thing here so I think things are a little clearer when it's mentioned first...